api_video.py

```
@@ -5,11 +5,31 @@
 import math
 import subprocess
 import re
+import os
+import shutil
 
 from requests_toolbelt import MultipartEncoder
 
 from . import config
 
+def downloadVideo(self, media_id, filename, media=False, path='videos/'):
+    if not media:
+        self.mediaInfo(media_id)
+        media = self.LastJson['items'][0]
+    filename = '{0}_{1}.mp4'.format(media['user']['username'], media_id) if not filename else '{0}.mp4'.format(filename)
+    #print filename
+    try:
+        clips = media['video_versions']
+    except:
```

Don't use bare except.
```except Exception``` at least